### DISCUSSION TITLE
package data is missing after commit the docker container

### DISCUSSION THREAD

**Discussion by JLUiceman** [verdaccio/verdaccio#2267] [Q&A] [answered]
I'm using verdaccio with docker; 
And i try to commit the docker container as a new docker image, after i loaded the image file in a new server, and run the new image, i find out the all the package data in path '/verdaccio/storage/data' is missing(the package data is exist in the old image)
Anyone can help or someone have the same problem?

**Comment by juanpicado** [comment, marked as answer]
please provide examples.

**Reply by JLUiceman** [reply]
sorry for the late reply, I just solved this problem by using '-v' option to bind mount storage to host while running the docker container.
And thx for your reply:)
